decidim-admin/app/packs/src/decidim/admin/external_domain_whitelist.js

```diff
@@ -0,0 +1,100 @@
+import AutoButtonsByPositionComponent from "./auto_buttons_by_position.component"
+import AutoLabelByPositionComponent from "./auto_label_by_position.component"
+import createDynamicFields from "./dynamic_fields.component"
+import createSortList from "./sort_list.component"
```

Comment:
I think we should change these import statements to absolute paths, e.g. `src/decidim/admin/auto_buttons_by_position.component`.